NetworkPkg/UefiPxeBcDxe/PxeBcDhcp4.c

```diff
@@ -1749,14 +1749,17 @@ PxeBcDhcp4Dora (
   AsciiPrint ("\n");

```

Comment:
Please drop the merge commit (i.e. rebase the existing commit onto the top of release/202405).